src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -27,7 +27,7 @@ public class PersonBuilder {
     private Email email;
     private Address address;
     private Set<Skill> skills;
-    private Set<Mod> mods;
+    private Set<Module> mods;
```

Comment:
Perhaps it would be better to make all the 'mods' become 'modules', to be consistent with the namings... I noticed this inconsistency in several other places too.